openapi/box.openapi.yaml

```
@@ -2043,4 +2136,71 @@ components:
+    CreateSshAccessRequest:
+      type: object
+      description: |
+        Optional body for SSH access creation. If omitted entirely, defaults apply
+        (root user, server-configured expiry).
+      properties:
+        unix_user:
+          type: string
+          nullable: true
+          description: Unix user account to use for the SSH session (e.g. "ubuntu", "root").
+          example: "ubuntu"
           description: Opaque token for next page
```

YAMLlint (1.38.0)

[error] duplication of key "description" in mapping

(key-duplicates)